plansys2:
  ros__parameters:
    use_real_time: false
    
planner_client:
  ros__parameters:
    plan_solver_timeout: 500.0  
planner:
  ros__parameters:
    plan_solver_timeout: 500.0  
    plan_solver_plugins: ["LLM"]
    POPF:
      plugin: "plansys2/POPFPlanSolver"
    LLM:
      plugin: "plansys2/LLMPlanSolver"
      python_command: "/home/jfisherr/cuarto/2c/plansis/plansys_ws/src/TFG/tfg-ia/tfg_langchain/get_plan.py"
      python_env: "/home/jfisherr/cuarto/2c/plansis/plansys_ws/src/TFG/tfg-ia/.venv/bin/python"
    TFD:
      plugin: "plansys2/TFDPlanSolver"
executor:
  ros__parameters:
    bt_builder_plugin: "SimpleBTBuilder"
    # bt_builder_plugin: "STNBTBuilder"
